2021-08-07 16:24:29,173 INFO    Thread-21 :8628 [wandb_setup.py:_flush():69] setting env: {'project': 'medici wandb test', 'entity': 'unchae', 'root_dir': 'C:\\Users\\medici\\파이썬베이스\\7-8월 데이콘', 'sweep_id': 'p0hwfbf7', 'run_id': 'hneiuwbl', 'sweep_param_path': 'C:\\Users\\medici\\파이썬베이스\\7-8월 데이콘\\wandb\\sweep-p0hwfbf7\\config-hneiuwbl.yaml'}
2021-08-07 16:24:29,174 INFO    Thread-21 :8628 [wandb_init.py:_log_setup():338] Logging user logs to C:\Users\medici\파이썬베이스\7-8월 데이콘\wandb\run-20210807_162429-hneiuwbl\logs\debug.log
2021-08-07 16:24:29,174 INFO    Thread-21 :8628 [wandb_init.py:_log_setup():339] Logging internal logs to C:\Users\medici\파이썬베이스\7-8월 데이콘\wandb\run-20210807_162429-hneiuwbl\logs\debug-internal.log
2021-08-07 16:24:29,175 INFO    Thread-21 :8628 [wandb_init.py:init():371] calling init triggers
2021-08-07 16:24:29,176 INFO    Thread-21 :8628 [wandb_init.py:init():376] wandb.init called with sweep_config: {'colsample_bytree': 0.6536476992793909, 'max_depth': 2, 'subsample': 0.764610425178516}
config: {}
2021-08-07 16:24:29,176 INFO    Thread-21 :8628 [wandb_init.py:init():420] starting backend
2021-08-07 16:24:29,176 INFO    Thread-21 :8628 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-08-07 16:24:29,180 INFO    Thread-21 :8628 [backend.py:ensure_launched():135] starting backend process...
2021-08-07 16:24:29,281 INFO    Thread-21 :8628 [backend.py:ensure_launched():139] started backend process with pid: 3448
2021-08-07 16:24:29,281 INFO    Thread-21 :8628 [wandb_init.py:init():425] backend started and connected
2021-08-07 16:24:29,292 INFO    Thread-21 :8628 [wandb_run.py:_config_callback():874] config_cb None None {'colsample_bytree': 0.6536476992793909, 'max_depth': 2, 'subsample': 0.764610425178516}
2021-08-07 16:24:29,293 INFO    Thread-21 :8628 [wandb_run.py:_label_probe_notebook():846] probe notebook
2021-08-07 16:24:29,293 INFO    Thread-21 :8628 [wandb_run.py:_label_probe_notebook():853] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-07 16:24:29,294 INFO    Thread-21 :8628 [wandb_init.py:init():473] updated telemetry
2021-08-07 16:24:29,295 INFO    Thread-21 :8628 [wandb_init.py:init():492] communicating current version
2021-08-07 16:24:31,456 INFO    Thread-21 :8628 [wandb_init.py:init():497] got version response 
2021-08-07 16:24:31,457 INFO    Thread-21 :8628 [wandb_init.py:init():505] communicating run to backend with 30 second timeout
2021-08-07 16:24:31,879 INFO    Thread-21 :8628 [wandb_init.py:init():530] starting run threads in backend
2021-08-07 16:24:36,894 INFO    Thread-21 :8628 [wandb_run.py:_console_start():1625] atexit reg
2021-08-07 16:24:36,894 INFO    Thread-21 :8628 [wandb_run.py:_redirect():1499] redirect: SettingsConsole.WRAP
2021-08-07 16:24:36,894 INFO    Thread-21 :8628 [wandb_run.py:_redirect():1536] Wrapping output streams.
2021-08-07 16:24:37,888 INFO    Thread-21 :8628 [wandb_run.py:_redirect():1560] Redirects installed.
2021-08-07 16:24:37,888 INFO    Thread-21 :8628 [wandb_init.py:init():555] run started, returning control to user process
2021-08-07 16:26:21,978 INFO    Thread-21 :8628 [wandb_run.py:finish():1295] finishing run unchae/medici wandb test/hneiuwbl
